src/formatDate.js

```diff
@@ -8,7 +8,67 @@
  * @returns {string}
  */
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const separator = date.replace(/\d/gi, '')[0];
+  const newSeparator = toFormat[toFormat.length - 1];
```

Comment:
Minor, but I don't quite get, why you don't use just `toFormat[3]`

Reply:
I don't know) 
Very strange behavior)